app/web/components/Navigation/ReportButton.tsx

```diff
@@ -145,77 +131,77 @@ export default function ReportButton({
           startIcon: classes.startIcon,
         }}
       >
-        {(!isResponsive || !isBelowMd) && REPORT}
+        {(!isResponsive || !isBelowMd) && t("report.label")}
       </Button>
       <Dialog
         aria-labelledby="bug-reporter"
         open={isOpen}
         onClose={handleClose}
       >
-        <DialogTitle id="bug-reporter">{REPORT}</DialogTitle>
+        <DialogTitle id="bug-reporter">{t("report.label")}</DialogTitle>
         {type === "initial" ? (
           <>
             <DialogContent>
               <Button
                 onClick={() => setType("bug")}
                 className={classes.typeButton}
               >
-                {REPORT_BUG_BUTTON}
+                {t("report.bug.button_label")}
               </Button>
               <Button
                 onClick={() => setType("content")}
                 className={classes.typeButton}
               >
-                {REPORT_CONTENT_BUTTON}
+                {t("report.content.button_label")}
               </Button>
             </DialogContent>
             <DialogActions>
               <Button
                 onClick={() => handleClose({}, "button")}
                 variant="outlined"
               >
-                {CANCEL}
+                {t("cancel")}
               </Button>
             </DialogActions>
           </>
         ) : type === "content" ? (
           <>
             <DialogContent>
               <Typography variant="body1" paragraph>
-                {REPORT_CONTENT_MESSAGE}
+                {t("cancel")}
```

Comment:
```suggestion
                {t("report.content.message")}
```
Since it's completely broken at the moment with the wrong text

Reply:
Whoops